### ISSUE TITLE
tabletActiveArea not set on single screen

### ISSUE THREAD

**Issue by jcpearson** [tb2097/wacom-gui#8]
When the prefs are written on a single screen system, tabletActiveArea is empty, so the xinput command line has the wrong syntax - e.g. 

```
xinput set-prop "Wacom Intuos4 6x9 stylus" --type=float "Coordinate Transformation Matrix"
```

i.e. it is missing the actual Matrix ... which gives errors when attempting to load the prefs

A simple fix could be:

```
--- ./wacom-gui/options.py.dist 2018-05-23 22:45:22.000000000 +0100
+++ ./wacom-gui/options.py      2018-05-24 14:39:13.605167721 +0100
@@ -175,10 +175,12 @@ class otherOptions(QtGui.QWidget):
 
     def getScreenArea(self):
         setCommands = []
+        if self.tabletActiveArea == "":
+            self.tabletActiveArea = "HEAD-0"
         for device in self.devices:
             if device != "pad" and device != 'touch':
                 if 'HEAD' in self.tabletActiveArea:
-                    setCommands.append("xsetwacom set \"%s %s\" MapToOutput %s" %
+                    setCommands.append("xsetwacom --set \"%s %s\" MapToOutput %s" %
                                        (self.deviceNames[device], device, self.tabletActiveArea))
                 else:
                     setCommands.append("xinput set-prop \"%s %s\" --type=float \"Coordinate Transformation Matrix\" %s"
```

(also change 'xsetwacom set' to 'xsetwacom --set' to be consistent with the other xsetwacom prefs)

**Comment by tb2097** [comment]
Fixed in V0.2.0e2

**Comment by jcpearson** [comment]
Thanks - works fine now
